types/jest/index.d.ts

```diff
@@ -28,6 +28,7 @@
 //                 Pawel Fajfer <https://github.com/pawfa>
 //                 Regev Brody <https://github.com/regevbr>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.8
```

Comment:
```suggestion
// TypeScript Version: 3.8
```

Reply:
See, I'd prefer that for consistency, but the readme says [it should have that word](https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-use-features-from-typescript-33-or-above) 🤔